packages/sui-theme/src/settings-compat-v7/_components.scss

```
@@ -122,13 +122,14 @@ $c-button-flat-icon-stroke: transparent !default;
 $bgc-button-flat-active: $c-gray !default;
 
 // Card
+$bdc-article-featured: $c-featured !default;
+$bd-article-featured: $bdw-l solid $bdc-article-featured !default;
```

The alphabetical order is the way my friend 😜

```suggestion
$bd-article-featured: $bdw-l solid $bdc-article-featured !default;
$bdc-article-featured: $c-featured !default;
```